docs/source/getting-started/installation.rst

```diff
@@ -43,9 +54,8 @@ or install from source:
   $ pip install ".[all]"
 
 
-.. note::
-
-    For Macs, macOS >= 10.15 is required to install SkyPilot. Apple Silicon-based devices (e.g. Apple M1) must run :code:`pip uninstall grpcio; conda install -c conda-forge grpcio=1.43.0` prior to installing SkyPilot.
+Docker Installation
+===================
```

Comment:
```suggestion
```
We probably do not want to encourage this very experimental mode, since the berkeleyskypilot/skypilot:latest image is likely very old (not connected to CI/CD).